Dockerfile

```diff
@@ -1,33 +1,36 @@
-FROM hexpm/elixir:1.10.3-erlang-22.3.3-debian-stretch-20200224
+###
+# Three-stage Dockerfile
+###
 
-WORKDIR /root
+# 1.) Get the Elixir dependencies within an Elixir container
+FROM hexpm/elixir:1.10.3-erlang-22.3.3-debian-stretch-20200224 as elixir-builder
```

Comment:
@ianwestcott we moved away from using that, because it doesn't pin the Erlang version behind the scenes. This has caused issues for us in the past where a point release of Erlang has broken some of our dependencies.

Reply:
Should we revert back to the 2-stage version of the `Dockerfile` then?

Reply:
No, just the switch back to the `hexpm/elixir` image. If you want to use Debian Buster, it looks like the latest is `1.10.3-erlang-22.3.3-debian-buster-20200224`

Reply:
@paulswartz oh yikes, good to know. There are a couple places where we're still using it, I'll follow up separately.